src/App.js

```diff
@@ -0,0 +1,214 @@
+import React, { Component } from 'react';
```

Comment:
For unit 2 and at SHIFT we use `standard JS` it makes for really clean javascript and the library can even fix files for you.  There is also a great VS code plugin that works with it. https://standardjs.com/